README.md

```diff
@@ -40,8 +40,8 @@ Chrome extensions
 
 2. Install [Lowcode Chrome Extension](https://github.com/iteria-ui/lowcode/tree/master/packages/browser-extension)
 
-- Go to /packages/browser-extension
-- `yarn install` to install dependencies.
+- Go to /packages/browser-extension folder in repository.
```

Comment:
you might want to use markdown hyperlink - something like [/packages/browser-extension
](/packages/browser-extension
)